### PR TITLE
Some more Prettier refactoring, removing unused things and moving propTypes

### PR DESCRIPTION
**Pull request by rupertotorres1**
Not really related to my ticket, but decided to refactor a couple more things.

### CODE REVIEW

File: client/src/components/FilterMenu.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+import { bool, func } from 'prop-types';
+import PlacesAutocomplete, { geocodeByAddress, getLatLng } from 'react-places-autocomplete';
 import '.././css/FilterMenu.css';
 
+const propTypes = {
+  visibility: bool.isRequired,
+  changeSalary: func.isRequired,
+  changeLocation: func.isRequired,
+};
+
```
Comment (rupertotorres1) [inline client/src/components/FilterMenu.js:11]: Moving propTypes to the top for visibility: https://github.com/airbnb/javascript/tree/master/react#ordering

File: client/src/components/FilterMenu.js [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
 class FilterMenu extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      isMenuVisible: false,
```
Comment (rupertotorres1) [inline client/src/components/FilterMenu.js:13]: Seems like this isn't used and we're already passing in a visibility prop anyway.

File: client/src/components/FilterMenu.js [hunk 3]
```diff
@@ -45,40 +47,40 @@ class FilterMenu extends Component {
...
+  };
 
   // source: https://github.com/hibiken/react-places-autocomplete
   renderFunc = ({ getInputProps, getSuggestionItemProps, suggestions }) => (
     <div className="autocomplete-root">
       <input {...getInputProps()} />
       <div className="autocomplete-dropdown-container">
         {suggestions.map(suggestion => (
-          <div {...getSuggestionItemProps(suggestion)}>
+          <div {...getSuggestionItemProps(suggestion)} key={suggestion.id}>
```
Comment (rupertotorres1) [inline client/src/components/FilterMenu.js:83]: I believe we should generally use a key when mapping over things, I think this works (tested briefly).

File: client/src/views/Navbar.js [hunk 4]
```diff
@@ -1,7 +1,11 @@
 import React, { Component } from 'react';
+import { func } from 'prop-types';
 import Dropdown from '../components/Dropdown';
 import '../css/NavBar.css';
-import fetchPostings from '../actions/postingsActions';
```
Comment (rupertotorres1) [inline client/src/views/Navbar.js]: Not used?

File: client/src/views/Navbar.js [hunk 5]
```diff
@@ -31,29 +35,7 @@ class Navbar extends Component {
     this.updateSearchCategory(this.searchCategories[1]);
   };
 
-  _searchJobs = async() => {
```
Comment (rupertotorres1) [inline client/src/views/Navbar.js]: Not used? I think we use the search handler in the props.

File: client/src/components/FilterMenu.js [hunk 6]
```diff
@@ -45,40 +47,40 @@ class FilterMenu extends Component {
...
 
   toggleSalary = () => {
     this.setState(prevState => ({
       isShowSalary: !prevState.isShowSalary,
     }));
-  }
+  };
 
-  onChangeSalary = (event) => {
+  onChangeSalary = event => {
```
Comment (frostyshadows) [inline client/src/components/FilterMenu.js]: I think we should keep the parentheses around `event` for coding standards consistency, and it's easier to read.

File: client/src/views/Navbar.js [hunk 7]
```diff
@@ -13,13 +17,13 @@ class Navbar extends Component {
     this.searchCategories = ['Position', 'Location'];
   }
 
-  updateSearchCategory = (newCategory) => {
+  updateSearchCategory = newCategory => {
```
Comment (frostyshadows) [inline client/src/views/Navbar.js]: could you keep the parentheses here as well

File: client/src/views/Navbar.js [hunk 8]
```diff
@@ -13,13 +17,13 @@ class Navbar extends Component {
...
     this.setState({
       searchCategory: newCategory,
     });
   };
 
-  updateSearchKeywords = (event) => {
+  updateSearchKeywords = event => {
```
Comment (frostyshadows) [inline client/src/views/Navbar.js]: and here

**Review by frostyshadows — COMMENTED** [review]
Looked good when I tried running it, left a few comments.

**Review by cowmanjoe — APPROVED** [review]
LGTM

**Pull request merged**